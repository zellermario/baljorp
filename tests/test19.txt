create SJ m 0

create Eszkimo e1 m
create Eszkimo e2 m
create RaketaAlkatresz r1 player e2
create RaketaAlkatresz r2 player e2
create RaketaAlkatresz r3 field m

digobject e1
pass e1