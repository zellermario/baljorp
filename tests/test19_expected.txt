Fields:
SJ m (0) - object: x
	neighbours: []

Buildings:

Actors:
> Eszkimo e1 (heat: 5) on m
  Inventory: [r3]
> Eszkimo e2 (heat: 5) on m
  Inventory: [r1, r2]